pkg/services/sqlstore/user.go

```diff
@@ -495,17 +495,19 @@ func (o byOrgName) Less(i, j int) bool {
 }
 
 func (ss *SQLStore) GetUserOrgList(ctx context.Context, query *models.GetUserOrgListQuery) error {
-	query.Result = make([]*models.UserOrgDTO, 0)
-	sess := x.Table("org_user")
-	sess.Join("INNER", "org", "org_user.org_id=org.id")
-	sess.Join("INNER", x.Dialect().Quote("user"), fmt.Sprintf("org_user.user_id=%s.id", x.Dialect().Quote("user")))
-	sess.Where("org_user.user_id=?", query.UserId)
-	sess.Where(notServiceAccountFilter(ss))
-	sess.Cols("org.name", "org_user.role", "org_user.org_id")
-	sess.OrderBy("org.name")
-	err := sess.Find(&query.Result)
-	sort.Sort(byOrgName(query.Result))
-	return err
+	return ss.WithDbSession(ctx, func(dbSess *DBSession) error {
+		query.Result = make([]*models.UserOrgDTO, 0)
+		sess := x.Table("org_user")
```

Comment:
Does this `x.` and the two below it on ln 502 need to be replaced with `dbSess` ?

Reply:
good catch